tests/tests.schema

```diff
@@ -3925,61 +3922,7 @@
           "$ref": "#/definitions/Microsoft.RegexRecognizer"
         },
         {
-          "$ref": "#/definitions/Microsoft.AgeEntityRecognizer"
```

Comment:
I'm curious why these entity recognizers shouldn't be in the schema?

Reply:
They were not removed---just reordered into sorted order.

Reply:
@chrimc62 they have not been reordered in this PR, they have been removed.  This is the definition of: Microsoft.IRecognizer:
https://github.com/microsoft/botbuilder-dotnet/blob/d1c7e93e173d71d15f119c4a5b08995fc1586078/tests/tests.schema#L3898

```json
"Microsoft.IRecognizer": {
      "title": "Microsoft recognizer",
      "description": "Components which derive from Recognizer class",
      "$role": "interface",
      "oneOf": [
        {
          "$ref": "#/definitions/Microsoft.CrossTrainedRecognizerSet"
        },
        {
          "$ref": "#/definitions/Microsoft.LuisRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.MultiLanguageRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.OrchestratorRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.QnAMakerRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.RecognizerSet"
        },
        {
          "$ref": "#/definitions/Microsoft.RegexRecognizer"
        },
        {
          "type": "string"
        }
      ]
    }
```

Should actually be:
```json
   "Microsoft.IRecognizer": {
      "title": "Microsoft recognizer",
      "description": "Components which derive from Recognizer class",
      "$role": "interface",
      "oneOf": [
        {
          "type": "string"
        },
        {
          "$ref": "#/definitions/Microsoft.OrchestratorRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.LuisRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.QnAMakerRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.CrossTrainedRecognizerSet"
        },
        {
          "$ref": "#/definitions/Microsoft.MultiLanguageRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.RecognizerSet"
        },
        {
          "$ref": "#/definitions/Microsoft.RegexRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.AgeEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.ChannelMentionEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.ConfirmationEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.CurrencyEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.DateTimeEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.DimensionEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.EmailEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.GuidEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.HashtagEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.IpEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.MentionEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.NumberEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.NumberRangeEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.OrdinalEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.PercentageEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.PhoneNumberEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.RegexEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.TemperatureEntityRecognizer"
        },
        {
          "$ref": "#/definitions/Microsoft.UrlEntityRecognizer"
        }
      ]
    }
```